packages/slate-react/src/components/editable.tsx

```diff
@@ -1049,6 +1053,37 @@ export const Editable = (props: EditableProps) => {
   )
 }
 
+/**
+ * The props that get passed to renderPlaceholder
+ */
+export type RenderPlaceholderProps = {
+  contentEditable: boolean
+  children?: React.ReactNode
+  ref: React.RefObject<any>
+}
```

Comment:
Can you also add an `attributes['data-slate-placeholder']: true` to that dictionary as well? That will be consistent with the others and allow people to see it clearly in DevTools as well as make simple tweaks to styles with overrides when necessary.